src/v2/Apps/PriceDatabase/Routes/MarketingLanding/__tests__/PriceDatabaseApp.jest.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import { MarketingLandingApp } from "../MarketingLandingApp"
+import { mount } from "enzyme"
+import { HeadProvider } from "react-head"
+
+describe("MarketingLandingApp", () => {
+  it("renders correct components", () => {
+    const wrapper = mount(
+      <HeadProvider>
```

Comment:
Note that we can use `MockBoot` which is like a simulated app environment (`Boot.tsx` is what brings in all of our shared contexts)